src/components/apps/launch/ParamGroups.js

```diff
@@ -123,28 +122,35 @@ const buildParamId = (baseId, paramIndex, type) => {
 /**
  * Form fields and info display for an app parameter group.
  */
-const ParamGroupForm = withI18N((props) => {
-    const classes = useStyles();
-
+function ParamGroupForm(props) {
     const {
         baseId,
         fieldName,
+        noOfGroups,
+        index,
         group,
         referenceGenomes,
         referenceGenomesLoading,
         startingPath,
     } = props;
+    const classes = useStyles();
+    const theme = useTheme();
+    const isMobile = useMediaQuery(theme.breakpoints.down("xs"));
 
     return (
-        <Accordion id={baseId} defaultExpanded>
-            <AccordionSummary
```

Comment:
I think we agreed in today's meeting to stick with Accordions for now, for the way they offset the parameter groups from each other, but we also wanted better styling for the group header/summary 👍

Reply:
This change is done. 
![image](https://user-images.githubusercontent.com/1250790/88836392-9a318600-d1a4-11ea-85fb-d631ada4e945.png)

![image](https://user-images.githubusercontent.com/1250790/88836419-a61d4800-d1a4-11ea-91d0-0c3d6a9261e4.png)

Reply:
I have used `info` color and contrast text. The section numbers are under the `group` title. If anyone has a feedback, please bring it up in next UX meeting.